Description: Add spl/arndale-spl.bin rule
Author: Ian Campbell <ijc@debian.org>

Index: u-boot/Makefile
===================================================================
--- u-boot.orig/Makefile
+++ u-boot/Makefile
@@ -2108,6 +2108,9 @@ spl/u-boot-spl: tools prepare $(if $(CON
 spl/sunxi-spl.bin: spl/u-boot-spl
 	@:
 
+spl/arndale-spl.bin: spl/u-boot-spl
+	@:
+
 spl/sunxi-spl-with-ecc.bin: spl/sunxi-spl.bin
 	@:
 
